include/verilated_vpi.cpp

```diff
@@ -494,10 +499,14 @@ class VerilatedVpiImp {
         bool called = false;
         typedef std::set<VerilatedVpioVar*> VpioVarSet;
         VpioVarSet update;  // set of objects to update after callbacks
-        const auto end = cbObjList.end();  // prevent looping over newly added elements
-        for (auto it = cbObjList.begin(); it != end;) {
+        if (cbObjList.empty()) { return called; }
```

Comment:
```suggestion
        if (cbObjList.empty()) return called;
```

Style nit - similar elsewhere.